docs/cdn_installation.md

```diff
@@ -90,6 +90,7 @@ The snippet below shows several configuration options with the body of the snipp
 | path | String | `/` | See https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent |
 | sameSite | Boolean | `true` | See https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent |
 | secure | Boolean | `true` | See https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent |
+| unique | Boolean | `false` | When this field is `false`, the session cookie name is `cwr_s`. When this field is true `true`, the session cookie name is `cwr_s_[AppMonitor Id]`.<br/><br/>Set this field to `true` when multiple AppMonitors will monitor the same page. For example, this might be the case if one AppMonitor is used for logged-in users, and a second AppMonitor is used for guest users.  |
```

Comment:
Is there any downside to always using the suffixed behavior?

Reply:
My reasoning for preferring `cwr_s` from a technical perspective is that the name takes up fewer bytes. My reasoning for preferring `cwr_s` from a non-technical perspective is that (1) it promotes setting the cookie at the appropriate domain and path, and (2) it makes it easier to document the names of the cookies for consent/transparency purposes.